include/osmium/util/file.hpp

```diff
@@ -222,6 +222,22 @@ namespace osmium {
             return static_cast<std::size_t>(offset);
         }
 
+        /**
+         * Set current offset into file.
+         *
+         * @param fd Open file descriptor.
+         * @param offset Desired absolute offset into the file
+         */
+        inline void file_seek(int fd, size_t offset) noexcept {
+#ifdef _MSC_VER
+            osmium::detail::disable_invalid_parameter_handler diph;
+            // https://msdn.microsoft.com/en-us/library/1yee101t.aspx
+            _lseeki64(fd, offset, SEEK_SET);
```

Comment:
We'll need a cast here (and below for the `lseek`) because `off_t` is a signed type and `size_t` is unsigned, to avoid warnings.